lib/sunrise/helper.rb

```diff
@@ -1,7 +1,36 @@
 module Sunrise
   module Helper
-    def html &block
-      # Implement me!
+    def html &code
+      instance_exec &code if block_given?
```

Comment:
Når nogen mixer dette modul ind leaker alle metoderne med ind i klassen/modulet. F.eks ville dette være muligt:
```ruby
class PersonComponent
  include Sunrise::Helper

  def full_name
    # Kalder metoden fra modulet
    add_attribute 
  end
end
```
Det kan være en godt at spørge sig selv om hvor meget af koden fra modulet man egentlig har lyst til at gøre tilgængelig for der mixer det ind. Det kan hurtigt blive en gang method clutter, hvis man mixer mange moduler ind, som hver har en stor bunke metoder.

Overvej i stedet at flytte alt logikken fra modulet, til en klasse, som du kan passe block'en der kommer ind i `#html` videre til. Så har du tingene helt adskilt, og det eneste man får når man mixer `Sunrise::Helper` ind er `#html`-metoden.

Reply:
Ahh, ofc! Det var nok det, Rubocop brokkede sig over så. Det skal jeg lige se, om jeg kan få til at virke. 👍